Dockerfile-LTS-Carbon

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.6
```

Comment:
Why it's not latest release `3.7.0` ?

Reply:
Hm IDK I was not updating that, but when I initially did this Dockerfile it was a recommended/showcased version of alpine for that version of node (8), perhaps we can update and see if it is ok 🤔  I'll give it a try in a bit and update here...

Reply:
Unfortunately, I was unable to build an image for 3.7 successfully, IDK why it fails but perhaps it needs a newer version of node (9 or 10) so leaving it as is.